agent/acl_endpoint.go

```diff
@@ -978,7 +1031,7 @@ func (s *HTTPHandlers) ACLLogout(resp http.ResponseWriter, req *http.Request) (i
 	s.parseToken(req, &args.Token)
 
 	if args.Token == "" {
-		return nil, acl.ErrNotFound
+		return nil, HTTPError{StatusCode: http.StatusUnauthorized, Reason: "Token supplied with request not found"}
```

Comment:
I wonder if it makes a difference to say "ACL system don't recognize the token supplied in the request".
For the following reasons:
1. "not found" could imply it never existed, but "recognize" implies, it might exist before, but we don't acknowledge it know, IMO.
2. Active tense, with the subject. Maybe they've used token from elsewhere, we don't know, but we identify it's our ACL doesn't know this one.
3. I assume the request is "log out with deleting this token" and the token is not found, then the token is included ("in") the request, not "with" the request? 
I could be wrong. LMK how you think.

Reply:
updated to "Supplied token does not exist"

Reply:
After discussion around the difference between the two token not found error messages:
1. "supplied token does not exist" from user scenario: User wants to read their own token.
2. "token supplied with request not found" from user scenario: user wants to destroy the token using itself.

We want to **keep both the same as "Supplied token does not exist"** for the following reasons:
1. Consul does not know any other information other than `we can't find the token` and `users are giving this particular request` therefore we can't provide more information for user to access the problem.
2. We cannot provide directions for the users in these two scenarios without making incorrect assumptions and potentially mislead the user.
4. "does not exist" should be sufficient information for the users to understand all the info we can provide.
5. Consistency in error messaging- user should realize both messages are coming from the same cause: Consul did not find this ACL token.